### PR TITLE
PRO-4248 - Caveats create demo env

### PR DESCRIPTION
**Pull request by Bishwin**
https://tools.hmcts.net/jira/browse/PRO-4248

### CODE REVIEW

File: infrastructure/demo.tfvars [hunk 1]
```diff
@@ -1,47 +1,44 @@
...
 packages_version = "3.0.0"
 
-probate_frontend_hostname = "probate-frontend-demo.service.core-compute-demo.internal"
-external_host_name = "probate.demo.platform.hmcts.net"
-outbound_proxy = ""
-
-probate_google_track_id = "UA-93598808-1"
-
-probate_redis_url = "betaPreProdprobatecache01.reform.hmcts.net"
+probate_business_service_url = "http://betaPreProdprobateApp01.reform.hmcts.net:4101/validate"
```
Comment (reddy1praveen) [inline infrastructure/demo.tfvars]: remove this

File: infrastructure/sandbox.tfvars [hunk 2]
```diff
@@ -1,47 +1,44 @@
...
 env = "sandbox"
+external_host_name = "probate-caveats.sandbox.platform.hmcts.net"
+
+//app settings var
 deployment_env = "test"
 
-vault_section = "test"
+feature_toggles_api_url = "http://rpe-feature-toggle-api-demo.service.core-compute-demo.internal"
+
+idam_service_api = "http://betadevbccidams2slb.reform.hmcts.net"
```
Comment (reddy1praveen) [inline infrastructure/sandbox.tfvars]: idam service host is chanfed

Comment (Bishwin) [reply infrastructure/sandbox.tfvars]: changed it to idamservicehost from PA frontend

Comment (Wivenhoe-HMCTS) [reply infrastructure/sandbox.tfvars]: idam_service_api changed to http://betadevbccidams2slb.reform.hmcts.net

File: infrastructure/demo.tfvars (hunk 1 above)
Comment (Wivenhoe-HMCTS) [reply infrastructure/demo.tfvars]: donee

**Pull request merged**